source/process/marketing-guidelines.rst

```diff
@@ -0,0 +1,65 @@
+============================================================
+Mattermost Marketing Content Guidelines
+============================================================
+
+How to write purposefully and persuasively 
+==============================================================
+
+In this document we set out some easy-to-follow guidelines about writing. Follow them, while also relying on your own judgment. Writing for blogs, newsletters, magazine articles and other forms of content can then become a joy rather than a chore. 
+
+Before we get into the guidelines themselves, let's consider the goal and objectives (why we think written communication is important in the first place) and a few principles (things to bear in mind when pursuing those goals).
+
+Goal
+---------------------------------------------------------
+
+We write marketing content in order to encourage organizations to bring the benefits of modern communication with Mattermost to our target audience.
```

Comment:
@jasonblais good point, propose `Success in marketing means more people in our target audience discover, understand, and unlock the benefits of Mattermost for their organizations through faster, larger, more effective deployments.`

^ so discovery leads to understanding leads to unlocking (via deployment)